contribute.py

```diff
@@ -85,6 +94,12 @@ def arguments():
                         to the repository. The link is accepted in SSH or HTTPS
                         format. For example: git@github.com:user/repo.git or
                         https://github.com/user/repo.git""")
+    parser.add_argument('-un', '--user-name', dest='user_name', type=str,
+                        required=False, help="""An optional user.name
```

Comment:
It might be better to rephrase somewhat like this: `Overrides user.name git config. If not specified, the global user.name is used.` The same applies to email.